Scripts/fn_respiratorEffects.sqf

```diff
@@ -0,0 +1,68 @@
+/*
+ * Author: Alganthe, TyroneMF
+ * Specified masks protect from a contamination zone while providing HUD/Sound effects.
+ * Requires a marker covering an area named "Contamination" for damage to take effect.
+ *
+ * Call from initPlayerLocal.sqf
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_player] call FUNC(respiratorEffects);
+ */
+#include "..\script_component.hpp"
+
+params ["_player"];
+
+GVAR(maskCounter) =  CBA_missionTime;
+GVAR(lastSoundRan) = CBA_missionTime;
+GVAR(oldGlasses) = "";
+[{
+    private _masks = [
+        "avon_ct12", // AVON FM12 Mod Respirators
+        "avon_ct12_strapless",
+        "avon_fm12",
+        "avon_fm12_strapless",
+        "avon_SF12",
+        "avon_SF12_strapless",
+        "G_AirPurifyingRespirator_01_F", // Contact DLC respirators
+        "G_AirPurifyingRespirator_02_black_F",
+        "G_AirPurifyingRespirator_02_olive_F",
+        "G_AirPurifyingRespirator_02_sand_F",
+        "G_RegulatorMask_F"
+    ];
+
+    private _goggles = goggles ACE_player;
+
+    if (_goggles in _masks) then {
```

Comment:
`in` is case-sensitive. To be safe, use:
```suggestion
    if ((toLower _goggles) in _masks) then {
```
and lower-case everything in `_masks` (look at other comment).